<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>My First App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>537</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="firstNameLineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>39</number>
            </property>
            <property name="placeholderText">
             <string>First Name max 40 characters</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="surnameLabel">
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="surnameLineEdit">
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>40</number>
            </property>
            <property name="placeholderText">
             <string> Surname max 40 characters</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="cPRNumberLabel">
            <property name="text">
             <string>CPR-number</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="cPRNumberLineEdit">
            <property name="maximumSize">
             <size>
              <width>97</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>CPR Number in form ######-####</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhHiddenText|Qt::ImhUrlCharactersOnly</set>
            </property>
            <property name="inputMask">
             <string>999999-9999</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="streetLabel">
            <property name="text">
             <string>Street</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="streetLineEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="streetNumberLabel">
            <property name="text">
             <string>Street Number</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="extLabel">
            <property name="text">
             <string>Ext</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="extLineEdit">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>9</number>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="zipCodeLabel">
            <property name="text">
             <string>Zip code</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="zipCodeLineEdit">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>9999</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="cityLabel">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="cityLineEdit">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="firstNameLabel">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="streetNumberLineEdit">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>NNNN</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonClear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Clear fields</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonOK">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>surnameLineEdit</tabstop>
  <tabstop>cPRNumberLineEdit</tabstop>
  <tabstop>streetLineEdit</tabstop>
  <tabstop>streetNumberLineEdit</tabstop>
  <tabstop>extLineEdit</tabstop>
  <tabstop>zipCodeLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
